source/ToSort/XSharp.VS/XSharp.VS.csproj

```diff
@@ -19,4 +19,8 @@
         <ProjectReference Include="..\XSharp\XSharp.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
```

Comment:
The package version should be updated in `Versions.props`, not here. Please remove this `ItemGroup`.